src/get-environments.ts

```diff
@@ -98,31 +96,7 @@ async function getOrganizationId(apiKeyCredentials: {
     await axios.get<{ id: string }[]>(`https://${ENV0_API_URL}/organizations`, {
       auth: apiKeyCredentials,
     })
-  ).data[0]?.id;
+  ).data[1]?.id;
```

Comment:
I'm not sure we would like to change it.. I guess you did for local development

Reply:
yes this is only for me locally 
297022c90327d0e8a8196a3043ee6ebb1f4d75f9